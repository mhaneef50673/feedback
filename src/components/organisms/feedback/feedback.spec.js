import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { Feedback } from './index';

describe('Feedback Component', () => {
  it('should render the component', () => {
    const tree = shallow(
      <Feedback />
    )
    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});