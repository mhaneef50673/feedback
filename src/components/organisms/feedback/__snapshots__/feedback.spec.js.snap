// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback Component should render the component 1`] = `
<Container
  fluid={false}
>
  <Row
    noGutters={false}
  >
    <Col>
      <FeedbackForm />
    </Col>
    <Col>
      <Connect(FeedbackTrend) />
    </Col>
  </Row>
  <Row
    noGutters={false}
  >
    <Col>
      <Connect(FeedbackComments) />
    </Col>
  </Row>
</Container>
`;
